contrib/docker_build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+uid=`id -u`
+gid=`id -g`
+
+version=$1
+[ -n "${version}" ] || version="latest"
+
+docker build \
+    --no-cache \
+    -t readthedocs/build-dev:${version} \
+    --build-arg uid=${uid} \
+    --build-arg gid=${gid} \
+    --build-arg label=${version} \
+    .
```

Comment:
To build these images we don't need to pass any context, this just makes the operation slower. You can just pass the dockerfile like
```suggestion
    -< Dockerfile
```

> If you use STDIN or specify a URL pointing to a plain text file, the system places the contents into a file called 
Dockerfile, and any -f, --file option is ignored. In this scenario, there is no context.

https://docs.docker.com/engine/reference/commandline/build/